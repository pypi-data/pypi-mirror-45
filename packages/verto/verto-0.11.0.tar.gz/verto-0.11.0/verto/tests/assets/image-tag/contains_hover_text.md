Some text before.

{image file-path="computer-studying-turing-test.png" hover-text="Lipsum" alt="placeholder image"}
