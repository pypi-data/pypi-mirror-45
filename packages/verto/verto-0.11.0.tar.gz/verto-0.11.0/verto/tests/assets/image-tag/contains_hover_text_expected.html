<p>Some text before.</p>
<div>
<img alt="placeholder image" class="" src="{% static 'computer-studying-turing-test.png' %}" title="Lipsum" />
</div>
